avail-server/src/main/java/com/avail/server/io/TransportAdapter.kt

```diff
@@ -87,4 +90,30 @@ interface TransportAdapter<T> : AutoCloseable
 	 *   A channel.
 	 */
 	fun sendClose(channel: AbstractTransportChannel<T>)
+
+	/**
+	 * A function that accepts a [DisconnectReason] and the underlying
```

Comment:
Revise this based on other changes made to `onChannelCloseAction` parameters and fields elsewhere, as mentioned above.